src/validation.cpp

```diff
@@ -2427,14 +2525,20 @@ QtumTransaction QtumTxConverter::createEthTX(const EthTransactionParams& etp, ui
     else{
         txEth = QtumTransaction(txBit.vout[nOut].nValue, etp.gasPrice, etp.gasLimit, etp.receiveAddress, etp.code, dev::u256(0));
     }
-    dev::Address sender(GetSenderAddress(txBit, view, blockTransactions));
+    dev::Address sender(GetSenderAddress(txBit, view, blockTransactions, (int)nOut));
     txEth.forceSender(sender);
     txEth.setHashWith(uintToh256(txBit.GetHash()));
     txEth.setNVout(nOut);
     txEth.setVersion(etp.version);
+    txEth.setRefundSender(refundSender);
 
     return txEth;
 }
+
+size_t QtumTxConverter::stackSize(size_t size){
```

Comment:
The name of this is somewhat misleading, "correctedStackSize" or something like that might be more clear

Reply:
Changed the method name to `correctedStackSize`